scripts/base/init-bare.zeek

```diff
@@ -5832,8 +5824,16 @@ export {
 		## A :zeek:see:`Telemetry::MetricOpts` record describing this metric.
 		opts: MetricOpts;
 
+		## The label names (also called dimensions) of the metric. When
+		## instantiating or working with concrete metrics, corresponding
+		## label values have to be provided. Examples of a label might
+		## be the protocol a general observation applies to, the
+		## directionality in a traffic flow, or protocol-specific
+		## context like a particular message type.
+		label_names: vector of string &default=vector();
```

Comment:
Yeah here's where you're losing me Tim — having both label names and label values in separate vectors doesn't look very practical. Wouldn't it be easier for this to be a `table[string] of string`?

Reply:
One problem with this is that tables can't have `&log` attributes, so we'll have to find another way to log them.

Reply:
Okay Tim, thanks. I was picturing an iteration over the table to populate the `Info` record. I'll fiddle with it some more and we can iterate again on just that aspect if really needed — better to get the other fixes in place now.